proposals/serverlessworkflow.md

```diff
@@ -0,0 +1,170 @@
+- a [landscape document](https://docs.google.com/spreadsheets/d/10rSQ8rMhYDgf_ib3n6kfzwEuoE88qr0amUPRxKbwVCk/edit#gid=0)
+  that lists well-known open-source and proprietary Serverless platforms
+  and tools
+- a set of recommended next steps for the WG, as part of the whitepaper:
+  - encourage more Serverless technology vendors and open source developers
+    to join the CNCF
+  - foster an open ecosystem by establishing interoperable APIs, in particular
+    around: Events, Deployments and Workflows
+  - provide additional education as needed
+
+In addition to CloudEvents specification, the Serverless WG has agreed to form a sub working group to work on
+the Serverless Workflow specification.
+
+Goals of the Serverless Workflow sub-group are to :
+
+- Standardize Serverless Workflow model and definition
+- Facilitate Serverless Workflow portability
+- Be completely vendor neutral
+- Support both stateless and stateful Serverless Workflow implementations
+- Supply a light-weight, human-readable, and embeddable format for describing serverless workflows
+
+Functional scope of the Serverless Workflow specification includes:
+
+- Define and orchestrate steps/states involved in a serverless application.
+- Define which functions are executed in each step.
+- Define which event or combination of events trigger function execution.
+- Define function execution behavior (sequential, parallel, etc).
+- Specify information filtering throughout the execution of the serverless workflow.
+- Define error conditions with retries.
+- Define correlation between events and workflow instances.
+- Specify workflow integration with third-party services.
+
+Here is diagram that illustrates the Serverless Workflow functional flow:
+
+<p align="center">
+<img src="https://github.com/cncf/wg-serverless/raw/master/workflow/spec/media/sample-serverless-workflow2.png" with="400px" height="260px" alt="Serverless Workflow Diagram"/>
+</p>
+
+You can see the Serverless Workflow specification examples
+[here](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/spec-examples.md).
+
+You can see the Serverless Workflow specification use-cases
+[here](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/spec-usecases.md)
+
+You can see the Serverless Workflow specification logo
+[here](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/media/logo-small-text.png).
+
+The work on this specification is currently being done within the
+CNCF's Serverless WG Workflow sub-group, but with the large growth of the specification,
+and the short-term plan for a release of our first milestone (v0.1), it would make
+sense for this work to be more formalized as a new sandbox project under the CNCF.
+
+The goals/roadmap of the project include:
+
+- finalize release v0.1.
+- keep improving specification to v1.0.
+- define requirements for implementation compliance.
+- define requirements for Serverless Workflow API/SDK.
```

Comment:
It might be good to include something about verification of the spec through implementations and testing, or something to that effect

Reply:
added